static/src/stylesheets/head.footballSnaps.scss

```diff
@@ -7,5 +7,6 @@
 @import 'module/football/_match-stat';
 @import 'module/football/_match-summary';
 @import 'module/football/_matches';
+/*@import 'module/survey/_dropdown';*/
```

Comment:
For some reason I can't import this - perhaps it's not a proper SCSS module? In any case, it seems to get included automatically.

I'll remove but wanted to ask about this before doing so.

Reply:
If it is already included in the page can we delete this commented out one?